examples/containers/run.sh

```diff
@@ -16,6 +16,15 @@ set -eu
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+COUCHBASE_SERVER_IMAGE=${COUCHBASE_SERVER_IMAGE:-couchbase/server:7.0.2}
+
+DOCKER_USER=${DOCKER_USER:-couchbase}
+DOCKER_TAG=${DOCKER_TAG:-v1}
+CMOS_IMAGE=${CMOS_IMAGE:-$DOCKER_USER/observability-stack:$DOCKER_TAG}
+
+# Ensure we built the container locally first
+make -C "${SCRIPT_DIR}/../.." container
```

Comment:
Would this cause problems if `make containers-oss` is called, which builds `container-oss` and then calls this entrypoint script which then builds `container`?

Reply:
Yeah this is why we need to switch over to the official release at some point. We should not be building anything in the examples really but developers should know what they want explicitly.
If you don't have SSH access then you cannot currently run the example but there's likely little point using this with the OSS version for now.

Reply:
Ideally the scripts are standalone so if people are doing things beforehand fine but the script wants to do the specific workflow it is set up for. Otherwise we end up with weird coupling between scripts that is hard to document or enforce.

Reply:
Yeah I forget the changes adding `example-containers-oss` were to my branch and not to main, will remove them in that case.

Reply:
I suppose I could remove it and ensure people do it that way via documentation.